l10_ir_accounting/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+On the employee form view you will have 3 fields,
+one for personnel_code,
+second for iran_iD_number,
+last for father_name.
```

Comment:
this module is not adding these fields

and we have partner_firstname module to add part of this

Reply:
For the HR module that will be added soon

Copied (deleted)